invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -563,6 +586,22 @@ def get_rights(self, obj):
                 entry["rightsUri"] = link
             serialized_rights.append(entry)
 
+        # Adding access_status information
+
+        access_status = obj.get("access", {}).get("status", "")
+        if access_status == "metadata-only":
+            access_status = "closed"
+
+        access_right_formatted = access_status.capitalize() + " Access"
+        rights_uri = "info:eu-repo/semantics/{}Access".format(access_status)
```

Comment:
Other string formatting in this file are using f-strings; should we use an f-string here too for consistency?